src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -180,7 +179,7 @@ object ProjectFortis extends App {
               "accessToken" -> System.getenv("FACEBOOK_AUTH_TOKEN"),
               "appId" -> System.getenv("FACEBOOK_APP_ID"),
               "appSecret" -> System.getenv("FACEBOOK_APP_SECRET"),
-              "pageId" -> "aljazeera"
+              "pageIds" -> "aljazeera,cnn,bloomberg"
```

Comment:
Nit: elsewhere ([example](https://github.com/CatalystCode/project-fortis-spark/blob/f609ef8c67f86a62a65ef9208074bdbe0bfb345b/src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala#L133)) we use the pipe character `|` to separate values.